src/loaders/data-sources/AnnDataSource.js

```diff
@@ -159,15 +165,28 @@ export default class AnnDataSource extends ZarrDataSource {
   }
 
   /**
-   * Class method for loading the cell names from obs.
-   * @returns {Promise} An promise for a zarr array containing the names.
+   * Class method for loading the obs index.
+   * @returns {Promise} An promise for a zarr array containing the indices.
    */
-  loadCellNames() {
-    if (this.cellNames) {
-      return this.cellNames;
+  loadObsIndex() {
```

Comment:
I don't think I mind having these named `loadCellNames` & `loadGeneNames` since for `AnnData` the obs/var are specific keys for Anndata and correspond to cell & gene. Is there a case for anndata where the `obs` and `var` keys in zarr correspnod to something different?

Reply:
Yes, the `obs` and `var` are general and can refer to 'anything" really that is an observation and variable.  Our Vitessce names are really what is wrong; for example, you can use `obs` to store, as Heath does, glomerulae masks and then mass-spec quantification in `var`.  But he has to use our `anndata-cells` loader to that so we are the ones introducing the naming mismatch, which is why I wanted to move that mismatch to as far away (and to as isolated) a location in the loaders as possible.  Perhaps one day we will update our terminology.

Reply:
I will add some comments in the `AnnDataSource` and `AnnDataLoader` explaining this, giving a reason for the encapsulation.

Reply:
I see. I'd still probably prefer avoiding the re-aliasing in the loaders (creating base `AnnDataLoader`), but I understand now why those method names are preferred on the `AnnDataSource`.